music_lib_search_tool/music_lib_search_tool/apps/music_collection_search/models.py

```diff
@@ -114,6 +134,17 @@ class Meta:
         db_table = 'song'
 
     def to_dict(self):
+        print(f'Create {self.title} Dict Format')
```

Comment:
We should remove all print statements on merge.

Reply:
Removed